pkg/controller/scheduling/scheduling.go

```diff
@@ -125,6 +132,21 @@ func PriorityClassName(req router.Request, computeClass *adminv1.ProjectComputeC
 	return computeClass.PriorityClassName, nil
 }
 
+// RuntimeClassName checks that a defined RuntimeClass exists and returns the name of it
+func RuntimeClassName(req router.Request, computeClass *adminv1.ProjectComputeClassInstance) (string, error) {
```

Comment:
You are exporting this function. I don't think it is used outside this package. If that's true, can you unexport (lowercase) it?

Reply:
Seems like you were copying the pattern of PriorityClassName. If the same is true for that one, please unexport it as well